src/Ocelot/DependencyInjection/ConfigurationBuilderExtensions.cs

```diff
@@ -84,5 +84,20 @@ public static IConfigurationBuilder AddOcelot(this IConfigurationBuilder builder
 
             return builder;

```

Comment:
You can reuse new defined method (line 89) in the overloaded `AddOcelot` method (line 40) like this:
```csharp
public static IConfigurationBuilder AddOcelot(this IConfigurationBuilder builder, string folder, IWebHostEnvironment env)
{
    // line 78
    return AddOcelot(builder, fileConfiguration, env); // merge block of lines 79-85
}
```

Reply:
@raman-m Reused the new method. Please review.